pages/docs/users/approvals/model-approvals.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-priority: 11000
+priority: 300
```

Comment:
I think we agreed priorities in a previous meeting, but the idea was mostly around having increasing priorities as you go down in layers.  So, everything at this layer is a 5 digit number.